src/components/WebexMeetingInfo/WebexMeetingInfo.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import WebexMeetingInfo, {formatMeetingTime} from './WebexMeetingInfo';
+
+jest.mock('../hooks/useMeeting');
+
+describe('Webex Meeting Info component', () => {
+  describe('snapshots', () => {
```

Comment:
```suggestion
  describe('snapshot', () => {
```
For proper grammar on the test description, but NBD